lib/ts/translation/translations.ts

```diff
@@ -3,5 +3,6 @@ export const defaultTranslationsCommon = {
         BRANDING_POWERED_BY_START: "Powered by ",
         BRANDING_POWERED_BY_END: "",
         SOMETHING_WENT_WRONG_ERROR: "Something went wrong. Please try again.",
+        SOMETHING_WENT_WRONG_ERROR_RELOAD: "Something went wrong. Please try again later or reload the page.",
```

Comment:
Do we use SOMETHING_WENT_WRONG_ERROR anywhere? Should we use SOMETHING_WENT_WRONG_ERROR_RELOAD in all places instead?

Reply:
We use that in some other places. There are many cases where reloading doesn't solve anything (and it also doesn't make the error message go away)